laplace-essay-on-probabilities.md

```diff
@@ -141,11 +141,11 @@ replaced by three pairs of balls invariably bound.
 
 When every case is favorable to an event, its probability becomes
 certainty, and its expression equal to unity. In this way certainty
-and probability are comparable, whatever essential difference there
-may be between the two states of the mind, when either a truth is
+and probability are comparable, although there is an essential difference
```

Comment:
Can you clarify for me: the original "sont comparables, quoiqu’il y ait une différence essentielle entre les deux états de l’esprit" seemed to use the subjunctive -- is that just a matter of French grammar vs. what I'm used to for the English subjunctive, and not meant to cast any doubt here on the status of the "différence essentielle"?

Reply:
That's right, French grammar pretty much mandates the subjunctive here. I don't read it as casting doubt on the différence essentielle as much as granting its existence while also kind of dismissing its relevance. At first, I parsed "quoiqu’il y ait une différence essentielle entre les deux états de l’esprit" as a phrase incise, but I think it's more like

```
Sous ce rapport, la certitude et la probabilité sont comparables, [quoiqu’il y ait une différence essentielle entre les deux états de l’esprit(, lorsqu’une vérité lui est rigoureusement démontrée, ou lorsqu’il aperçoit encore une petite source d’erreur)].
```

where the the rest of the sentence shows examples for the difference between certainty (a rigorous demonstration) and probability (there exists a small error).